Added templates for issues/features

Jon0505 opened the pull request:
Added templates for issues and features to resolve #3062. The feature template is a mess right now since I didn't know what you guys wanted for it, but I copied the format from #3062 for the issue template. I can also add templates for pull requests. I just need to know what to put on them.

Jon0505 committed:
Added templates for issues/features

Jon0505 committed:
Changed quote to bullet list

FedUpWith-Tech commented:
Oh this is cool, I had no idea github did this- do you have any suggestions for the format? I know the formatting isn't perfect but I just wanted to include the average information I've seen needed to get adequate support or to debug bugs.

If you have any suggestions I'd love to hear them or please commit them to this branch and if they look good I'll update the template on the open issue

FedUpWith-Tech commented:
RE: PR template, that would be a better question for @mikeprimm I personally would just include the section from the README.md (make sure to fix the typos that haven't been merged yet)  about PR's and the guidelines. I don't know if there has been or is a standard for PR's I'm no a dev by any means I just committed to maintain the github for the foreseeable future

Jon0505 commented:
I could copy over my personal PR template if you want. For the issue template, I could add a field for screenshots and a few check boxes at the bottom which indicate "I have looked at all other issues and this is not a duplicate", "I have been able to replicate this", "My instructions are clear for replicating this bug", etc.

FedUpWith-Tech committed:
Added checkboxes

I added the checkboxes that I feel like would be helpful and added a spot for screenshots, I don't want to overcrowd the bug report and I don't feel the need to collect all the possible data I *might* need on first contact

FedUpWith-Tech committed:
Me being dumb

forgot how to use the preview screen to make sure the formatting was correct :/
